tests/testthat/test-OTHER-create-track-hub-jointpeaks.R

```diff
@@ -0,0 +1,94 @@
+library(testthat)
+library(PeakSegPipeline)
+library(data.table)
+context("create track hub with joint peaks")
+test.data.dir <- file.path(Sys.getenv("HOME"), "PeakSegPipeline-test")
+non.integer.dir <- file.path(test.data.dir, "non-integer")
+demo.dir <- file.path(test.data.dir, "input")
+hub.txt <- file.path(demo.dir, "hub.txt")
+trackDb.txt <- file.path(demo.dir, "trackDb.txt")
+### Download bigWig files from github.
+bigWig.part.vec <- c(
+  "kidney/MS002201"
+)
+joint_peaks.bedGraph.file <- file.path(
+  demo.dir, "samples", bigWig.part.vec[1], "joint_peaks.bedGraph")
```

Comment:
hi can you please remove [1] here to avoid confusion (bigWig.part.vec is length 1)

Reply:
Hi, I also update these changes as well